diff --git a/sources/stat.c b/sources/stat.c
index 02efbdd..77044e3 100644
--- a/sources/stat.c
+++ b/sources/stat.c
@@ -1,11 +1,12 @@
 #include <stdlib.h>
 #include <mint/osbind.h>
 #include <errno.h>
-#include <ext.h>
+#include <sys/stat.h>
 #include "lib.h"
 
 #define X_MAGIC   0x601a
 
+#if 0
 #define S_IFMT    0170000  // bit mask for the file type bit fields
 #define S_IFSOCK  0140000  // socket
 #define S_IFLNK   0120000  // symbolic link
@@ -29,6 +30,7 @@
 #define S_IROTH   00004    // others have read permission
 #define S_IWOTH   00002    // others have write permission
 #define S_IXOTH   00001    // others have execute permission
+#endif
 
 int stat(const char *path, struct stat *buff)
 {
